src/node_api.h

```diff
@@ -112,6 +112,8 @@ NAPI_EXTERN napi_status
 napi_get_last_error_info(napi_env env,
                          const napi_extended_error_info** result);
 
+NAPI_EXTERN void napi_fatal_exception(napi_env env);
```

Comment:
I think I would prefer it if you could pass in the exception here, like the existing Node APIs do – you should be able to get a currently pending exception with `napi_get_and_clear_last_exception`.

Actually passing the exception along to Node’s fatal exception handling seems orthogonal to that?

Reply:
orthogonal? i'm ok with added a napi_value that is used as the exception yes

Reply:
Mh, sorry – [orthogonal](https://en.wikipedia.org/wiki/Orthogonality_(programming)) ~= independent … basically, I’d suggest that we provide users with the smallest pieces that make sense to be exposed in the API, rather than exposing a function that actually does two operations masquerading as one (getting the last exception + passing it to the fatal exception handler)